apis/nucleus/src/components/Cell.jsx

```diff
@@ -290,17 +290,36 @@ const Cell = forwardRef(
     const [selections] = useObjectSelections(app, model);
     const [hovering, setHover] = useState(false);
     const hoveringDebouncer = useRef({ enter: null, leave: null });
+    const [bgColor, setBgColor] = useState(undefined);
     const focusHandler = useRef({
       focusToolbarButton(last) {
         // eslint-disable-next-line react/no-this-in-sfc
         this.emit(last ? 'focus_toolbar_last' : 'focus_toolbar_first');
       },
     });
 
+    const resolveBgColor = () => {
+      const bgComp = layout && layout.components ? layout.components.find((comp) => comp.key === 'general') : null;
+
+      if (bgComp && bgComp.bgColor && bgComp.bgColor.useColorExpression) {
+        if (bgComp.bgColor.urlColor) {
+          bgComp.bgColor.urlColor = { color: bgComp.bgColor.urlColor };
+        }
+        return bgComp.bgColor.urlColor ? halo.public.theme.getColorPickerColor(bgComp.bgColor.urlColor) : undefined;
+      }
+      if (bgComp && bgComp.bgColor && !bgComp.bgColor.useColorExpression)
+        return bgComp.bgColor.cpColor ? halo.public.theme.getColorPickerColor(bgComp.bgColor.cpColor) : undefined;
```

Comment:
This is incorrect I think?
The property should be:
```
components[N] = {
  key: "general",
  bgColor: {
    color: { ...  } // support for 'none'  
  },
  bgImage: {
    mode: none, media, expression
    position: top-left etc...
    sizing: original, alwaysFit, alwaysFill, stretch, fitWidth, fitHeight
    mediaUrl: { qStaticContentUrlDef...}
    expressionUrl: { qStringExpression... }
  }
}

```
Should be `color` not `cpColor`.